### PR TITLE
Update Budget to compute actual and budget for wkfl2

### PR DESCRIPTION
**Pull request by jefmoura**
## Purpose
When a budget is added to a program associated with a project budget module, the total summed up budget should be shown in the project dashboard.

Related ticket: TolaDataV2/[#677](https://github.com/toladata/TolaDataV2/issues/677)

### CODE REVIEW

File: workflow/models.py [hunk 1]
```diff
@@ -1386,10 +1386,27 @@ class Budget(models.Model):
...
+        if not self.create_date:
+            if self.workflowlevel2:
+                wflvl2 = self.workflowlevel2
+                wflvl2.total_estimated_budget += self.proposed_value
+                wflvl2.actual_cost += self.actual_value
+                wflvl2.save()
             self.create_date = timezone.now()
+        elif self.workflowlevel2:
+            wflvl2 = self.workflowlevel2
+            old_budget = self.__class__.objects.get(id=self.id)
```
Comment (Menda) [inline workflow/models.py]: Would not it be better remove the redundant code inside the condition `if not self.create_date:` and place here an `if`?

And to get the old budget you can use a try, catch ObjectDoesNotExists.

Comment (jefmoura) [reply workflow/models.py]: I got your point but it doesn't raise `ObjectDoesNotExists`, it raises `AttributeError`.

Comment (jefmoura) [reply workflow/models.py]: Ok. you're talking about the `old_budget`. Sorry. hehe

Comment (jefmoura) [reply workflow/models.py]: Done @Menda

**Pull request merged**